presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcClient.java

```diff
@@ -59,6 +60,16 @@ default void abortReadConnection(Connection connection)
     PreparedStatement buildSql(Connection connection, JdbcSplit split, List<JdbcColumnHandle> columnHandles)
             throws SQLException;
 
+    void addColumn(JdbcTableHandle handle, ColumnMetadata column);
+
+    void dropColumn(JdbcTableHandle handle, JdbcColumnHandle column);
+
+    void renameColumn(JdbcTableHandle handle, JdbcColumnHandle jdbcColumn, String target);
```

Comment:
We might want to change `target` to `newColumnName` to be consistent with `renameTable` (if you rename here, also change all implementations)